CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.5.0] - 2022-08-11
```

Comment:
why 1.5.0? what happened to v1.3.0 and v1.4.0?

Reply:
Aligning with the version of CAPA. We've jumped from 1.2.0 to 1.5.0.

Reply:
I don't think it's the way we decided to go about this? I don't think we follow this approach on the other capi apps.